.github/workflows/nightly-tests.yml

```diff
@@ -16,6 +16,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      # Check out terra-helmfile repo
+      - uses: actions/checkout@v2
```

Comment:
It is unfortunate that you have to check out the entire repo to access two files.

Reply:
I have put up a new GitHub action `checkout-helm-versions` to do sparse check out of the two directories relevant to helm versions.